src/requirements/reqs.json

```diff
@@ -48,13 +48,6 @@
                     "name": "CALS Credits",
                     "description": "55 CALS credits are required for graduation. CALS credits include all courses from departments within CALS and courses offered in Applied Economics and Management, Biological Sciences, Biology & Society, Earth and Atmospheric Sciences, Information Science, Nutritional Science, and The Department of Statistics and Data Science.",
                     "source": "https://cals.cornell.edu/undergraduate-students/student-services/degree-requirements/graduation-requirements",
-                    "search": ["acadGroup"],
```

Comment:
out of curiosity, did you write a script to delete all of search and includes attributes or did u do it manually?

Reply:
Manually. The json still has something that I don't fully understand, so I manually deleted everything useless that I know for sure.